src/main/java/com/coveo/pushapiclient/DocumentUpdateService.java

```diff
@@ -0,0 +1,20 @@
+package com.coveo.pushapiclient;
+
+public class DocumentUpdateService {
+
+    public DocumentUpdateService addOrUpdateDocument(DocumentBuilder document) {
+        throw new java.lang.UnsupportedOperationException("TODO: To implement");
+    }
+
+    public DocumentUpdateService addOrUpdateDocument(PartialUpdateDocument document) {
+        throw new java.lang.UnsupportedOperationException("TODO: To implement");
```

Comment:
@Spuffynism and @toussaints, do you know what the status of partial update feature is longer term? Are we keeping it as-is? Changing how it works? Removing it?

Leaning towards leaving this mode out the first version of this sdk if we're not sure,